deploy/ui/form.json

```diff
@@ -107,26 +107,64 @@
           ]
         },
         {
-          "label": "Admin user",
-          "type": "fieldGroup",
+          "label": "Admin user configuration",
+          "type": "dropDown",
           "required": false,
-          "jsonPath": "$.spec.commonConfig",
+          "jsonPath": "$.spec.commonConfig.AdminUser",
           "visible": true,
+          "default": "default",
+          "options": [
+            {
+              "value": "secret",
+              "label": "Secret configuration"
+            },
+            {
+              "value": "default",
+              "label": "Username and Password configuration"
+            }
+          ],
           "fields": [
             {
-              "label": "Username",
-              "default": "adminUser",
+              "label": "Username and password to create a kie-admin-credentials secret",
```

Comment:
Credentials to create the Secret?

Reply:
Was the old label when the idea was to create a secret with the credentials provided, fixed